src/app/containers/StoryPromo/IndexAlsos/index.jsx

```diff
@@ -75,17 +75,26 @@ const IndexAlsosContainer = ({ alsoItems, script, service, dir }) => {
     <IndexAlsos offScreenText={relatedContent} data-e2e="index-alsos">
       <IndexAlsosWrapper>
         {alsoItems.slice(0, MAX_NUM_INDEX_ALSOS).map(item => {
-          const { id, cpsType, mediaType } = item;
+          const { cpsType, mediaType } = item;
+
+          const aresId = pathOr(null, ['id'], item);
+          const promoId = pathOr(null, ['uri'], item);
+          const id = aresId || promoId;
```

Comment:
`id` is used as the react component key, is it worth us having an addition check to ensure `id` is not null before passing it to the component. That might be overkill so am happy to leave it as is.

Reply:
It's now a required propType again.